content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -365,6 +365,40 @@ This applies only to significant, user-visible behaviors which impact the
 correctness of applications running on Kubernetes or that impact the
 administration of Kubernetes clusters, and which are being removed entirely.
 
+The exception to the above rule is **feature-gates**. Feature gates are key=value 
+pairs that enable users to enable/disable experimental features.
+Features can be in Alpha, Beta or GA stage.
+
+Feature gates are intended to cover the development of a feature - they are not 
+intended to be long-term APIs. As such, they are expected to be deprecated and 
+removed after a feature becomes GA or is dropped. 
+As a user of feature gates pay atention to deprecation announcements for feature-gates 
+when one of the following scenarios apply:
+
+  * The feature become GA - is enabled by default.
+  * The feature is a candidate to be dropped.
+  * The feature is a candidate to become a configurable option, this happens when the
+     old behavior is needed by users. The new option is introduced as soon as we
+     identify the need and it follows the feature lifecycle as well as feature
+     deprecation rules.
+
+In some cases, the old behavior requires a considerable amount of time before it can 
+be removed. In these cases the feature gate's behavior can be retained until the old 
+behavior is removed, at which point the feature gate can be deprecated as above.  
+
+Feature gates are not versioned in the same way as the previously discussed components,
+therefore the rules for deprecation are as follows:
+
+**Rule #8a: Feature gates must announce deprecation when the corresponding feature 
```

Comment:
Can these just be one rule?

```
** Rule #8: Feature gates must be deprecated when the features they control undergo the following lifecycle changes:
  * **Beta -> GA: 6 months or 2 releases (whichever is longer)**
  * **Beta -> EOL: 3 months or 1 release (whichever is longer)**
  * **Alpha -> EOL: 0 releases**
```

Reply:
@thockin 
I merged the rule as below. 
"Rule #8: Feature gates must announce deprecation when the corresponding feature 
they control transitions a lifecycle stage as below. They must function as long as the 
old feature is not deprecated for no less than:"
Note that I emphasize that we announce the gate's deprecation on life cycle change, however your comment expresses that the gate must be deprecated on life cycle change. Just wanted to confirm with you if I should change the meaning of the rule to "must be deprecated on life cycle change".

Reply:
Nit: the gates don't "announce deprecation", they "[are, get, become] deprecated".  The gates are not the subject of the sentence, but the object.

The phrase "as long as the old feature is not deprecated for no less than" is very awkward - double-negatives are hard to parse.  I'm not sure what it means.  Can you phrase it in the affirmative?